client/src/pages/Photo.js

```diff
@@ -65,6 +65,7 @@ class PhotoPage extends Component {
   };
 
   render() {
+    console.log(this.state.file);
```

Comment:
If testing is done, we can have this removed

Reply:
Yes, I'll remove the line.